### DISCUSSION TITLE
pip install mock Collecting mock   Could not find a version that satisfies the requirement mock (from versions: ) No matching distribution found for mock

### DISCUSSION THREAD

**Discussion by shru9289** [apache/incubator-mxnet#14861] [Q&A]
Note: Providing complete information in the most concise form is the best way to get help. This issue template serves as the checklist for essential information to most of the technical issues and bug reports. For non-technical issues and feature requests, feel free to present the information in what you believe is the best form.

For Q & A and discussion, please start a discussion thread at https://discuss.mxnet.io 

## Description
I am trying to run pip install mock
i get this error
pip install mock
Collecting mock
  Could not find a version that satisfies the requirement mock (from versions: )
No matching distribution found for mock

, i even tried to upgrade pip but it doesn't work
Not sure what's the issue i have python 2.7 version with me

## Environment info (Required)
Mac 

Package used (Python/R/Scala/Julia):
Ptyhon 2.7

[a comment by an automated account is not shown]

**Comment by lanking520** [comment]
@shru9289 This is a pip install problem, please try to upgrade your pip version or follow the instruction here to install mock:
https://pypi.org/project/mock/0.8.0/
